AndrewTehan/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './cli'
+require './rainfall_statistics'
+
+def run_cli
+  Cli.new.run_cli
```

Comment:
`Cli.new.run` is a quite readable naming, `Cli.new.run_CLI` is redundant a bit to me